bundles/org.openhab.core.io.rest.auth/src/main/java/org/openhab/core/io/rest/auth/internal/ExpiringUserSecurityContextCache.java

```diff
@@ -47,46 +50,65 @@ protected boolean removeEldestEntry(Map.@Nullable Entry<String, Entry> eldest) {
         };
     }
 
-    synchronized @Nullable UserSecurityContext get(String key) {
+    synchronized @Nullable SecurityContext get(@Nullable String key) throws AuthenticationException {
```

Comment:
Why allow null as a key?

Reply:
Afair, the previous code always made sure that the cacheKey had a value - so there should be no need for the check.